azure-functions/acbs-function/mappings/facility/facility-loan-amend.js

```diff
@@ -4,16 +4,28 @@ const helpers = require('./helpers');
 
 const { FACILITY } = CONSTANTS;
 
+/**
+ * Processes amendments to a facility loan record.
+ *
+ * @param {Object} amendments - The amendments to be applied.
```

Comment:
These should be lower case `object` I believe (all of these)

Reply:
Merged under PR #3600